### DISCUSSION TITLE
Calendar do PO-Ui com Datetime que gera a data e a hora?

### DISCUSSION THREAD

**Discussion by HorselessName** [po-ui/po-angular#1826] [Angular]
Olá, boa tarde. Estou montando um formulário de chamados que possui SLA, usando o Dynamic Forms.
Por possuir SLA eu precisava registrar as datas e as horas de início e fim do chamado...

Lendo a documentação eu vi que o `datetime` possui suport para a ISO-8601 pra formato de data e hora, e eu fiz assim:

```
{
      property: 'sla_parada',
      
      // Título do Input
      label: "Data e Hora de Parada",

      // ##### Definições de Data e Hora #####
      // Format: ISO-8601 --> yyyy-mm-ddThh:mm:ss+|-hh:mm
      // Exemplo: 2015-01-01T00:00:00
      type: "datetime",
      format: "dd/mm/yyyy hh:mm:ss",

      // Definir o Min e Max values com base no SLA de atendimento!
      // Formato: Ano - Mes - Dia
      minValue: "2023-01-01 00:00:00",
      maxValue: "2025-01-01 00:00:00",
      errorMessage: 'A data permitida é entre 2023 e 2025 apenas.',

      // Validador de Data - Debug/Log
      validate: this.onChangeField.bind(this),

      gridColumns: 6,
      order: 3
    }
```

Mas o calendário só está permitindo eu selecionar a data e não consigo setar a hora... 
Alguém teria alguma dica? 🥲

**Comment by anliben** [comment]
Bom Dia @HorselessName, tudo bem?

Obrigado por reportar, em breve iremos analisar e retornar uma resposta.

Attr.

**Comment by anliben** [comment]
Boa Tarde @HorselessName, tudo bem?

Nosso componente `po-calendar` não serve para captura de hora.

Attr.
